source/utilities.tex

```diff
@@ -5818,6 +5879,10 @@
 \tcode{is_copy_constructible_v<$\tcode{T}_i$>} is \tcode{true} for all $i$.
 If \tcode{is_trivially_copy_constructible_v<$\tcode{T}_i$>}
 is \tcode{true} for all $i$, this constructor is trivial.
+The exception specification is equivalent to the logical AND of
+%FIXME: The paper puts the "AND" in math font and I have no idea
+%what markup this is supposed to be in the standard.
```

Comment:
```suggestion
The exception specification is equivalent to the logical \logop{and} of
```

It's not math font in the paper, it's smallcaps. It should be styled exactly like the _Throws:_ element of `variant(variant&&)` four paragraphs later.